allauth/headless/internal/restkit/response.py

```diff
@@ -0,0 +1,28 @@
+from django.http import JsonResponse
+from django.utils.cache import add_never_cache_headers
+
+
+class APIResponse(JsonResponse):
+    def __init__(self, request, errors=None, data=None, meta=None, status=200):
+        d = {"status": status}
```

Comment:
Small aside: Is it useful to include the status in the response body? Usually the http client library would return the status code alongside the response body. e.g. 

```
response.status
```

Instead of 

```
response.data.status
```

Reply:
This is done to help out with Javascript, where we typically have this:

```
fetch(url)
    .then((response) => {
      return response.json();
    })
    .then((data) => {
      // here, response is not available, so we cannot access the status.
    })
```

... or:

```
async function fetchFoo(url) {
   return await fetch(url).json()
}
```

In both cases the `response` is gone.

Reply:
Got it, thanks for the clarification! 😄